### ISSUE TITLE
Provide feedback when there's no eth1 connection

### ISSUE THREAD

**Issue by paulhauner** [flex-dapps/lighthouse-client-ui#13]
Lighthouse users can disable syncing via eth1 (by not supplying `--eth1` or `--staking`). It would be nice if the GUI could tell users "you don't have eth1 enabled and you'll never sync" instead of showing a 0% progress or other error. However, there is no API endpoint for this, so this issue is blocked.

**Comment by daniel-flex** [comment]
Agree. When the flag is not enabled it returns an error message when hitting /lighthouse/eth1/syncing, so could use that.

**Comment by paulhauner** [comment]
> When the flag is not enabled it returns an error message when hitting /lighthouse/eth1/syncing

Oh yeah, that's right. That's all the information we need then.

Could we perhaps add the following text if you get that specific error?

> The beacon node needs to be restarted with the `--eth1` flag.

And we should make it red like an error, since it wont get fixed without them doing something.

**Comment by daniel-flex** [comment]
need designs around this - tagging design

**Comment by daniel-flex** [comment]
![Screen Shot 2020-10-31 at 9 53 58 am](https://user-images.githubusercontent.com/73456391/97763455-c7088700-1b5f-11eb-8ae6-4e2340ba1a12.png)
![Screen Shot 2020-10-31 at 9 54 15 am](https://user-images.githubusercontent.com/73456391/97763460-ca037780-1b5f-11eb-9ff8-c62faef0b071.png)

**Comment by qnou** [comment]
@daniel-flex So the tagging system is supposed to be propagated exactly where the text shows up in the card on the bottom left. Please view the prototype update for example.
